packages/components/badge/src/Badge/Badge.styles.ts

```diff
@@ -83,9 +76,10 @@ export const getBadgeStyles = () => ({
   badgeIcon: css({
     flexShrink: 0,
   }),
-  badgeText: css({
-    whiteSpace: 'nowrap',
-    overflow: 'hidden',
-    textOverflow: 'ellipsis',
-  }),
+  badgeText: ({ size }: { size: BadgeSize }) =>
+    css({
+      color: 'currentcolor',
+      lineHeight: 'inherit',
+      fontSize: size === 'small' && '0.688rem',
```

Comment:
Is this magic number cleared with @fabe? I would prefer not to use it if we can.

Reply:
This is because for the small variant we need 11px for font-size, and we don't have that on the tokens, we could create a token for it, but not sure if it's used in other places, @fabe will we use it in more places? Or should we fallback to the 12px token?

Reply:
Let's fall back to the 12px token then. I think it doesn't change the height of the element, so we can go for that!